workers/data_refinery_workers/processors/transcriptome_index.py

```diff
@@ -215,30 +228,68 @@ def _zip_index(job_context: Dict) -> Dict:
     only be a single file along with compressing the size of the file
     during storage.
     """
-    temp_post_path = job_context["gtf_file"].get_temp_post_path(job_context["job_dir_prefix"])
+
+    stamp = str(timezone.now().timestamp()).split('.')[0]
+    job_context["computed_archive"] = job_context['base_file_path'] + '/' + job_context['organism_with_size'] + "_" + stamp + '.tar.gz'
+
     try:
-        with tarfile.open(temp_post_path, "w:gz") as tar:
+        with tarfile.open(job_context["computed_archive"], "w:gz") as tar:
             tar.add(job_context["output_dir"],
                     arcname=os.path.basename(job_context["output_dir"]))
     except:
         logger.exception("Exception caught while zipping index directory %s",
                          temp_post_path,
-                         processor_job=job_context["job_id"],
-                         batch=job_context["batches"][0].id)
-
-        job_context["gtf_file"].remove_temp_directory(job_context["job_dir_prefix"])
-
+                         processor_job=job_context["job_id"]
+                         )
         failure_template = "Exception caught while zipping index directory {}"
         job_context["job"].failure_reason = failure_template.format(temp_post_path)
         job_context["success"] = False
+        try:
+            os.remove(job_context["computed_archive"])
+        except Exception:
+            pass
         return job_context
 
-    job_context["files_to_upload"] = [job_context["gtf_file"]]
+    job_context["files_to_upload"] = [job_context["computed_archive"]]
     job_context["success"] = True
     return job_context
 
+def _populate_index_object(job_context: Dict) -> Dict:
+    """ """
+
+    result = ComputationalResult()
+    result.command_executed = job_context["salmon_formatted_command"] # No op!
+    result.is_ccdl = True
+    result.system_version = __version__
+    result.time_start = job_context["time_start"]
+    result.time_end = job_context["time_end"]
+    result.save()
+
+    computed_file = ComputedFile()
+    computed_file.absolute_file_path = job_context["computed_archive"]
+    computed_file.file_name = os.path.split(job_context["computed_archive"])[-1]
+    computed_file.calculate_sha1()
+    computed_file.calculate_size()
+    computed_file.result = result
+    #computed_file.sync_to_s3(S3_BUCKET_NAME, computed_file.sha1 + "_" + computed_file.filename)
+    # TODO here: delete local file after S3 sync
+    computed_file.save()
+
+    organism_object = Organism.get_object_for_name(job_context['organism_name'])
+    index_object = OrganismIndex()
+    index_object.organism = organism_object
+    index_object.version = "XXX" # XXX: I don't know how this is tracked
```

Comment:
I made an issue so we don't just have a dangling unknown in the code: https://github.com/AlexsLemonade/refinebio/issues/158